llvm/lib/Target/PowerPC/PPCInstr64Bit.td

```diff
@@ -1457,7 +1457,7 @@ def LQ   : DQForm_RTp5_RA17_MEM<56, 0,
                                 "lq $RTp, $addr", IIC_LdStLQ,
                                 []>,
                                 RegConstraint<"@earlyclobber $RTp">,
-                                isPPC64;
+                                isPPC64, MemOp;
```

Comment:
This could move into `DQForm_RTp5_RA17_MEM` - this is the only use of that class.